App.js

```diff
@@ -31,30 +31,38 @@ import LoginPage from './components/auth/loginPage';
 import SignupPage from './components/auth/signupPage';
 
 import fakeUser from './data/profileData.js';
-import data from './data/data';
 const Tab = createBottomTabNavigator();
 
 export default function App() {
   // Determine user has light/dark theme on phone
   let phoneTheme = true;
   const colorScheme = Appearance.getColorScheme();
-  if (colorScheme === 'dark') {
+  if (colorScheme === 'light') {
     phoneTheme = false;
```

Comment:
I'm not sure I understand the need for both phoneTheme and isDarkTheme. If it works for now that's fine, but unless there's a particular reason, it could stand to be tidied up.

Reply:
So what is going on here with color scheme is checking to see if the phone has an internal preference toward dark themes. It is an option to have your phone default everything to dark theme, and this is a check to see if that preference is selected. isDarkTheme, on the other hand, is our app's monitor of whether dark or light theme is chosen. It allows a user to select dark or light outside of their phone's internal preference